### PR TITLE
updates and tests

### PR DESCRIPTION
**Pull request by njayp**
- helm install/uninstall action
- general fixes
- test workflow

### CODE REVIEW

**Comment by knlambert** [comment]
Any news on this one ? :D

**Comment by njayp** [comment]
> Any news on this one ? :D

I got it to build. Now i just need to get my tests to pass. I have an odd issue where core isn't propagating my secret

File: .github/workflows/integration-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        with:
+          telepresence_api_key: ${{ secrets.TELEPRESENCE_API_KEY }}
+
+      # we need a demo cluster
+      - uses: azure/setup-kubectl@v3
+        with:
+          version: "v1.25.3"
+        id: kubectl
+      - name: Provision Kubeception cluster
+        uses: datawire/infra-actions/provision-cluster@v0.2.0
```
Comment (shepz) [inline .github/workflows/integration-tests.yaml:32]: is there a reason not to use the last version ?

File: .github/workflows/integration-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        uses: ./intercept
+        with:
+          service_name: dataprocessingservice
+          service_port: 3000:3000
+          http_header: "x-telepresence-intercept-test=dataprocessingservice" # Custom HTTP header name and value that will identify traffic desired to go to the local service instace.
+          print_logs: true # Flag to instruct the action to print out Telepresence logs and export an artifact with them
+      - name: Test Intercept with Curl
+        shell: bash
+        run: |
+          sleep 3
```
Comment (shepz) [inline .github/workflows/integration-tests.yaml:71]: can we do something better than a sleep ?

**Review by shepz — COMMENTED** [review]
Does the README is up-to-late with those changes ?

File: .github/workflows/integration-tests.yaml (hunk 1 above)
Comment (njayp) [reply .github/workflows/integration-tests.yaml:32]: not really, i just copied from our integration tests w/o checking

File: .github/workflows/integration-tests.yaml (hunk 2 above)
Comment (njayp) [reply .github/workflows/integration-tests.yaml:71]: i dont think so. we could have a retry, but thats basically the same thing 🤷

**Review by shepz — APPROVED** [review]

**Pull request merged**